app/uk/gov/hmrc/fhregistrationfrontend/services/KeyStoreService.scala

```diff
@@ -19,28 +19,26 @@ package uk.gov.hmrc.fhregistrationfrontend.services
 import com.google.inject.{ImplementedBy, Singleton}
 import uk.gov.hmrc.fhregistrationfrontend.config.KeySessionCache
 import uk.gov.hmrc.http.HeaderCarrier
-import uk.gov.hmrc.http.cache.client.CacheMap
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 @Singleton
 class KeyStoreServiceImpl extends KeyStoreService {
-  val id = "fhdds-summary-input"
 
-  override def fetchAndGetEntry()(implicit hc: HeaderCarrier): Future[Option[String]]= {
-    sessionCache.fetchAndGetEntry[String](id)
-  }
-
-  override def save(o: String)(implicit hc: HeaderCarrier): Future[CacheMap] = sessionCache.cache(id, o)
+  val SummaryForPrintKey = "fhdds-summary-input"
 
   private val sessionCache = KeySessionCache
 
+  override def saveSummaryFormPrint(o: String)(implicit hc: HeaderCarrier): Future[_] = sessionCache.cache(SummaryForPrintKey, o)
```

Comment:
Should we be discarding the type associated to the Future computation? I'm not sure that we should. Will accept it for now but will review later when we look at the overall design and implementation.